olp-cpp-sdk-authentication/src/AuthorizeRequest.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "olp/authentication/AuthorizeRequest.h"
+
+#include <sstream>
+
+namespace olp {
+namespace authentication {
+
+std::string AuthorizeRequest::CreateKey() const {
```

Comment:
The method is non-trivial and it'd be nice to have a few unit tests for it.

Reply:
Please look https://github.com/heremaps/here-olp-sdk-cpp/pull/792/files/b04c47be5011cbe66ba5fb8379e729b4f4191bf8#diff-d1d1293083a502f525306c203f4bc75f, there is some checks for CreateKey

Reply:
There's just one check. It doesn't check the `if (GetContractId())` branch or `AND` case.

Reply:
File was updated https://github.com/heremaps/here-olp-sdk-cpp/pull/792/commits/b78a32acf765d415c043697596f75e533eeda2bd.